### DISCUSSION TITLE
ADC DMA  library

### DISCUSSION THREAD

**Discussion by JingtaiChen** [stm32duino/Arduino_Core_STM32#1875] [General]
Because the library does not support Arduino's API, ADC is slow. Can you develop a high-speed ADC library for beginners?eg：myADC

**Comment by fpistm** [comment]
Why you said the library does not support Arduino's API?

About ADC you can override the analog API:
https://www.stm32duino.com/viewtopic.php?t=110

**Comment by JingtaiChen** [comment]
I read the link you sent. I mean, can you make it easier to develop a library file and call it directly?

**Comment by fpistm** [comment]
There is already an issue open for this but currently, no time for this.
This is a community project any contributions are welcome.
https://github.com/stm32duino/Arduino_Core_STM32/issues/5
